db/migrate/20190806155914_create_alert_images.rb

```diff
@@ -0,0 +1,11 @@
+class CreateAlertImages < ActiveRecord::Migration
+  def change
+    create_table :alert_images do |t|
+      t.belongs_to :stolen_record, null: false, index: true, foreign_key: true
+      t.boolean :current, null: false, default: true
```

Comment:
1. I'd prefer not to have default true on booleans
2. Rather than `current` as a boolean, I feel like recording `retired_at` and then making a dynamic method for `current?` adds some value without adding much extra work

Reply:
Updated in 9ab375e